packages/manager/apps/pci-object-storage/src/pages/object-storage/create/_components/useOrderFunnel.hook.tsx

```diff
@@ -41,6 +41,8 @@ export function useOrderFunnel({
   const { t } = useTranslation('pci-object-storage/order-funnel');
   const orderFunnelFormSchema = createOrderFunnelFormSchema(t);
 
+  const defaultReplicationRegion = regions?.[0].name;
```

Comment:
In case we have no regions
```suggestion
  const defaultReplicationRegion = regions?.[0]?.name;
```